projects/systelab-login/src/lib/role-directives/forbidden-roles.directive.spec.ts

```diff
@@ -63,7 +63,7 @@ describe('ForbiddenRolesDirective', () => {
 				]
 		})
 			.compileComponents();
-	}));
+	});
 
 	beforeEach(() => {
```

Comment:
I have the same doubt as before about adding or not the async on this beforeEach. Thanks!

Reply:
The TestBed.compileComponents method asynchronously compiles all components configured in the testing module.
Please see https://angular.io/guide/testing-components-scenarios#the-async-beforeeach